client/src/api/v2/files.ts

```diff
@@ -1,20 +1,21 @@
 import { useState, useEffect, useCallback } from 'react';
 
-/**
- * @typedef {object} LogFile
- * @property {string} fileName  Log file name
- * @property {string} url       URL to download file
- */
+interface LogFile {
+  /** Log file name */
+  fileName: string;
+  /** URL to download file */
+  url: string;
+}
 
 /**
  * Get list of log files
  *
- * @returns {Promise<LogFile[]>} List of log files
+ * @returns List of log files
```

Comment:
JSDoc complains that we don't document the return type here. What to do? Obviously it would be redundant, I could disable the lint but then our JS code isn't required to have it. (maybe that's okay though if we're moving to TS)

Reply:
I think we can just disable the lint rule. All the exisiting JS has the jsdoc types already, and everything new will be typescript anyways

Reply:
Having some trouble disabling the JSdoc warnings that appear when running the dashboard. According to https://www.digitalocean.com/community/tutorials/react-linting-react#customizing-create-react-app-and-eslint, we can't configure eslint unless we eject from create react app, which I'm guessing we don't want to do. Any ideas?

Reply:
You can disable the rule in client/.eslintrc

Reply:
I tried doing this but for some reason the linter warnings still appear when running `yarn start`. Unless something is completely wrong with my config...